docs/content/sftp.md

```diff
@@ -578,6 +578,20 @@ Properties:
     - "~/.ssh/known_hosts"
         - Use OpenSSH's known_hosts file.
 
+#### --sftp-host-key-algorithms
```

Comment:
This part will be autogenerated from the definition of the option in the .go file - and overwritten. You currently have different description here and in there, so you should make sure the description you want to use are in the .go file (Help property), and then you can remove the change here in the .md.